### PR TITLE
chore: add more E2E tests for self-assessment

### PR DESCRIPTION
**Pull request by dmorand**
## Description

Add more E2E tests for self-assessment + daily CI enrollment.

## Checklist

- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines 
- [ ] All relevant PR sections are populated, irrelevant ones are removed

### CODE REVIEW

File: integration-tests/interactions/bot/daily_ci_enroll/registration_complete.jinja [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+[
+    {
+        "text": "Thanks. I have recorded all your answers in your personal profile."
+    },
+    {
+        "text": "Your registration to the daily check-in is now completed."
+    },
+    {
+        "text": "I will check-in with you everyday around this time to see how you're doing, monitor your symptoms and direct you to the right resources."
+    },
```
Comment (NetBKarine) [inline integration-tests/interactions/bot/daily_ci_enroll/registration_complete.jinja:12]: Is there a way to separate these messages and concatenate them after? These first 3 are always sent after enrollment, but the messages after depend on where we were before enrollment (suspect, returning or tested positive flow). There are many cases like this related to self-isolation and home-assistance sub-flows. If we have to copy them for each flow and something changes we'll have to change them in many places. It wouldn't be dramatic though, only sub-optimal.

File: integration-tests/interactions/bot/daily_ci_enroll/thank_you.jinja [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% import 'bot/macros.jinja' as bot %}
+
+[
+    {
+        "text": "Thanks and take care of yourself!"
+    }
+]
```
Comment (NetBKarine) [inline integration-tests/interactions/bot/daily_ci_enroll/thank_you.jinja:7]: This thank you message is generic, it should probably be at the root of the bot folder.

Comment (dmorand) [reply integration-tests/interactions/bot/daily_ci_enroll/thank_you.jinja:7]: Got it!

File: integration-tests/interactions/bot/daily_ci_enroll/registration_complete.jinja (hunk 1 above)
Comment (dmorand) [reply integration-tests/interactions/bot/daily_ci_enroll/registration_complete.jinja:12]: Yes, it bothered me too. It should be possible to extract them using Jinja macros. I'll figure something cute in another PR.

**Pull request merged**